prime-angular-frontend/src/app/modules/site-registration/site-registration.routes.ts

```diff
@@ -69,6 +78,18 @@ export class SiteRoutes {
     ];
   }
 
+  // Used to indicate the routes and order for sites for editing
+  public static editSiteRegistrationRouteOrder(): string[] {
```

Comment:
Could be `editRegistrationRouteAccess` which could include `remote-user`.  The method comment would need a small update indicating access not order, and a short comment above `remote-user` on why it was included to the list as a child route of `remote-users`.